assignments/objects.js

```diff
@@ -19,46 +20,125 @@ const example = {
 
 // Write your intern objects here:
 
+const michael = {
+    "id": 0,
+    "name": "Michael",
+    "email": "email@email.com",
+    "gender": "M"
+  }
+  const Mitzi = {
```

Comment:
I noticed you capitalized the names here on the assignment. While it does not affect how your code runs, it is considered best practice to let variables and objects be named in lowercase (except in the case of Classes and Components such as in React). Just something to keep in mind :+1: